### PR TITLE
feat: modal waiting cortex

### PR DESCRIPTION
**Pull request by urmauur**
## Describe Your Changes

- Update style modal witing cortex

Screenshot 2024-08-08 at 14 11 01

Screenshot 2024-08-08 at 14 10 54

## Fixes Issues

- #3292 
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: web/containers/WaitingCortexModal/index.tsx [hunk 1]
```diff
@@ -35,29 +43,53 @@ const WaitingForCortexModal: React.FC = () => {
...
+                {`Hmm, Jan's taking longer than usual to start up...`}
+              </h1>
+              <p className="mt-[2px] text-[hsla(var(--text-secondary))]">
+                API Server starting at{' '}
+                <a
+                  href={`${host}/api`}
+                  target="_blank"
+                  className="text-[hsla(var(--app-link))] hover:underline"
+                >
+                  {host}/api
```
Comment (Van-QA) [inline web/containers/WaitingCortexModal/index.tsx:61]: hi @urmauur, can you help me correct this? the URL should be:
- [url]/api
but the current out put is
- [URL]/v1/api :x: 
image

**Review by Van-QA — DISMISSED** [review]

File: web/containers/WaitingCortexModal/index.tsx (hunk 1 above)
Comment (urmauur) [reply web/containers/WaitingCortexModal/index.tsx:61]: @Van-QA updated!

Comment (louis-jan) [reply web/containers/WaitingCortexModal/index.tsx:61]: Nice, thanks @Van-QA @urmauur

File: web/containers/WaitingCortexModal/index.tsx [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
+  const [janDataFolderPath, setJanDataFolderPath] = useAtom(
+    janDataFolderPathAtom
+  )
 
   const checkSystemAlive = useCallback(async () => {
     setWaitingForCortex(!(await isSystemAlive()))
   }, [setWaitingForCortex, isSystemAlive])
 
+  const getJanDataFolderPath = useCallback(async () => {
+    setJanDataFolderPath(await window?.electronAPI.homePath())
```
Comment (louis-jan) [inline web/containers/WaitingCortexModal/index.tsx:27]: homePath is not correct btw, since we support relocate jan data folder

**Review by louis-jan — APPROVED** [review]

**Pull request merged**